templates/bridgeserver2.yaml

```diff
@@ -137,6 +152,57 @@ Parameters:
 Conditions:
   CreateDevResources: !Equals [ !Ref BridgeEnv, dev ]
 Resources:
+  DBCluster:
```

Comment:
Sorry I forgot about this but I think it would be better to create a separate CFN template for the database so that it's de-coupled from the application resources.  We do that for the agora DB resources, https://github.com/Sage-Bionetworks/Agora-infra/tree/develop/templates  

Otherwise templates in this PR looks good.